kernel/component-manager/src/main/java/org/sakaiproject/component/impl/SakaiContextLoader.java

```diff
@@ -68,7 +68,14 @@
     @Override
     public WebApplicationContext initWebApplicationContext(ServletContext servletContext) throws BeansException {
 
-        ConfigurableWebApplicationContext cwac = (ConfigurableWebApplicationContext) super.initWebApplicationContext(servletContext);
+        ConfigurableWebApplicationContext cwac = null;
```

Comment:
This was done to keep only one ContextLoader even when adding a Spring Boot app like Rubrics. You can check App.java file to see how SakaiApplicationContext is set as parent of Spring Boot's own loader.

java.lang.IllegalStateException: Cannot initialize context because there is already a root application context present - check whether you have multiple ContextLoader* definitions in your web.xml!
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:296)
	at org.sakaiproject.component.impl.SakaiContextLoader.initWebApplicationContext(SakaiContextLoader.java:71)